assignments/homework/sebastian/HW 10-13.py

```diff
@@ -0,0 +1,53 @@
+#HW 10----
+
+%matplotlib inline
+import matplotlib.pyplot as plt
+plt.rc('image', origin='lower')
+plt.rc('figure', figsize=(10, 6))
+
+from astropy.convolution import Gaussian2DKernel
+from astropy.io import fits
+
+gauss = Gaussian2DKernel(3)
+
+ISSA = fits.getdata('data/ISSA_100_LMC.fits')
+plt.imshow(convolve(ISSA, gauss))
+
+plt.imshow(scipy_convolve(ISSA, gauss.array))
+plt.imshow(convolve(gaia_map, gauss))
+
+#HW 12----
+
+%matplotlib inline
+import matplotlib.pyplot as plt
+plt.rc('image', origin='lower')
+plt.rc('figure', figsize=(10, 6))
+
+from photutils.datasets import load_star_image
+star_image = load_star_image()
+star_image
+
+from astropy.stats import median_absolute_deviation
+median_absolute_deviation(star_image.data)
```

Comment:
The exercise was called for getting the MAD for a sigma clipped image, the sigma clip step has been missed out here. All you need is the one line we did in class too, to get a `clipped_image`